contrib/dist/release_checklist_issue_template

```diff
@@ -0,0 +1,102 @@
+## Make a perfect tarball
+
+- [ ] Update the major,minor,release,greek version numbers in `VERSION`
+- [ ] Update the `c:r:a` shared library version number(s) in `VERSION` [per the GNU Libtool shared library version number rules](https://www.gnu.org/software/libtool/manual/html_node/Updating-version-info.html)
+   - **NOTE:** It may be helpful to use a command like `git checkout BRANCH; git pull --rebase; git log --stat --topo-order --decorate TAG_FROM_PREVIOUS_RELEASE..HEAD` to examine the Git logs and see what has changed.
+   - [ ] *IF RELEVANT:* If this is a new backwards-compatible release on a single branch (i.e., this is `vx.y.z` where `z>1`), you probably want to examine `git log --stat --no-merges last_release_tag..this_branch_name` to see what source code files have changed (which directly impacts how to increment the `c:r:a` values).
+  - [ ] *IF RELEVANT:* If this is a new release series (e.g., `vx.y.0`), set `r` to 0 and increase `c` values by 10 compared to the first release in the prior series (i.e., `vx.(y-1).0` or `v(x-1).0.0`, as relevant).
+  - [ ] *IF RELEVANT:* If this is a new backwards-compatible release series (i.e., `vx.y.0`, where `y>1`), set`a` values to 10 so that the shared libraries will be ABI compatible with the prior release series.
+  - [ ] *IF RELEVANT:* If this is a new backwards-**IN**compatible release series (i.e., `vx.0.0`), set `a` values to 0 so that there is no possibility of users accidentally mixing shared library versions.
```

Comment:
Do these instructions match those in the official libtool documentation? I have the following:
```
# These versions are maintained in accordance with the "Library
# Interface Versions" chapter from the GNU Libtool documentation:
#
# - If the library source code has changed at all since the last
#   update, then increment revision (`c:r:a' becomes `c:r+1:a').
#
# - If any interfaces have been added, removed, or changed since
#   the last update, increment current, and set revision to 0.
#
# - If any interfaces have been added since the last public release,
#   then increment age.
#
# - If any interfaces have been removed since the last public release,
#   then set age to 0.
#
```